contracts/shared/Roles.sol

```diff
@@ -1,5 +1,46 @@
 //SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.13;
 
+// Admin roles
 bytes32 constant DEFAULT_ADMIN_ROLE = 0x00;
-bytes32 constant MINTER_ROLE = keccak256("Minter");
+bytes32 constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
```

Comment:
Can we write a little bit about the difference between these two here?

Reply:
Essentially  `DEFAULT_ADMIN_ROLE` can grant and revoke any role, while `ADMIN_ROLE` just have controll over high level functions, like setting a controller or whitelisting an attribute.

Maybe should call it `MANAGER_ROLE` or something else? Admin is kinda of standard I think